etc/rucio.cfg.template

```diff
@@ -205,3 +206,6 @@ usercert = /opt/rucio/etc/usercert_with_key.pem
 [credentials]
 gcs = /opt/rucio/etc/google-cloud-storage-test.json
 signature_lifetime = 3600
+
+[undertaker]
```

Comment:
can this go into the core/config module instead of the rucio.cfg?

Reply:
yes, already done but will submit the new commit after PR #1950 got merged